Anguandia/quickcredit
# Add user verification function to UI

#### What does this PR do?

  Enable UI place user verification requests to server and appropriately
  render returned response

#### Description of Task to be completed?

  - Add fetch request for user verification
  - Add fetch request URL path to verify user menu item anchor href and
  point to the user's list(client.html)
  - Add function to render a list of users from a given collection array
  to select  the target loan from
  - Display user status in appropriate colors
  - Modify and rename item detail display function to accommodate user
  objects
  - Modify users javascript include file for users HTML page
  - Add general update fetch function

#### How should this be manually tested?

  - Visit https://anguandia.github.io/quickcredit/ and sign in with
  email admin@mail.com and password admin
  - Navigate to the verify user page
  - Select a user with status unverified and at the foot of the page
  rendered in the select action section, select verify, then submit
  - Notice the status change to verified
  - Login as admin from a different machine and click on clients; The
  user previously verified should stay so

#### What are the relevant pivotal tracker stories?

  [#166771579 #166722279 #166726332 #166726468]
